.github/workflows/occlum_sgx.yml

```
@@ -0,0 +1,40 @@
+name: CC kbc build CI
+on:
+  push:
+    paths:
+      - 'src/kbc_modules/cc_kbc/attesster/sgx_occlum'
+      - '.github/workflows/occlum_sgx.yml'
+  pull_request:
+    paths:
+      - 'src/kbc_modules/cc_kbc/attesster/sgx_occlum'
+      - '.github/workflows/occlum_sgx.yml'
+  create:
+    paths:
+      - 'src/kbc_modules/cc_kbc/attesster/sgx_occlum'
+      - '.github/workflows/occlum_sgx.yml'
+
+jobs:
+  build-and-run-occlum:
+    runs-on: self-hosted
+    container:
+      image: occlum/occlum:latest-ubuntu20.04
+      options: --device /dev/sgx/enclave --device /dev/sgx/provision
```

does the runner use some old driver or can you go with:
```suggestion
      options: --device /dev/sgx_enclave --device /dev/sgx_provision
```

---

I found this in occlum's document https://github.com/occlum/occlum#how-to-use, step 3. I manually create the soft link `/dev/sgx/enclave` from `/dev/sgx_enclave`. I'm not sure whether it is for occlum? cc @qzheng527

---

Occlum uses SGX SDK underneath and the latest SDK versions support the standard in-tree kernel provided device nodes. The symlinks are not needed and to stay future proof, `/dev/sgx_enclave` path is preferred.